commit 0ea74b4de34a12396fe3790590007aa50fcb5d45
Author: Kees Cook <kees@kernel.org>
Date:   Tue Jan 30 15:42:26 2024 -0800

    MAINTAINERS: Add UBSAN section
    
    The kernel hardening efforts have continued to depend more and more
    heavily on UBSAN, so make an actual MAINTAINERS entry for it.
    
    Cc: Andrey Ryabinin <ryabinin.a.a@gmail.com>
    Acked-by: Andrey Konovalov <andreyknvl@gmail.com>
    Acked-by: Marco Elver <elver@google.com>
    Signed-off-by: Kees Cook <keescook@chromium.org>

diff --git a/MAINTAINERS b/MAINTAINERS
index 8999497011a2..d0df728734c1 100644
--- a/MAINTAINERS
+++ b/MAINTAINERS
@@ -22473,6 +22473,23 @@ F:	Documentation/block/ublk.rst
 F:	drivers/block/ublk_drv.c
 F:	include/uapi/linux/ublk_cmd.h
 
+UBSAN
+M:	Kees Cook <keescook@chromium.org>
+R:	Marco Elver <elver@google.com>
+R:	Andrey Konovalov <andreyknvl@gmail.com>
+R:	Andrey Ryabinin <ryabinin.a.a@gmail.com>
+L:	kasan-dev@googlegroups.com
+L:	linux-hardening@vger.kernel.org
+S:	Supported
+T:	git git://git.kernel.org/pub/scm/linux/kernel/git/kees/linux.git for-next/hardening
+F:	Documentation/dev-tools/ubsan.rst
+F:	include/linux/ubsan.h
+F:	lib/Kconfig.ubsan
+F:	lib/test_ubsan.c
+F:	lib/ubsan.c
+F:	scripts/Makefile.ubsan
+K:	\bARCH_HAS_UBSAN\b
+
 UCLINUX (M68KNOMMU AND COLDFIRE)
 M:	Greg Ungerer <gerg@linux-m68k.org>
 L:	linux-m68k@lists.linux-m68k.org